### PR TITLE
[all hosts] (Support) Following Teams support page format

### PR DESCRIPTION
**Pull request by AlexJerabek**
As per @TristanD-MSFT's request, we want our support page to look like [that of Teams](https://learn.microsoft.com/en-us/microsoftteams/platform/feedback). This PR merges our information with that format.

### CODE REVIEW

**Review by TristanD-MSFT — APPROVED** [review]
Looks good, thanks for the quick adjustments!

File: docs/resources/resources-links-help.md [hunk 1]
```diff
@@ -1,25 +1,76 @@
...
 # Office Add-ins additional resources
 
 These resources provide additional information and support for developing Office Add-ins.
 
-|**Resource type**|**Guidance**|
+## Product support and service issues
+
+Use **Azure Admin Center** or **Microsoft 365 Admin Center** for any **business-critical issues** that need SLA-based support.
+
+| **Product support** | **Contact** |
```
Comment (ElizabethSamuel-MSFT) [inline docs/resources/resources-links-help.md:16]: I don't think you need to include the bold formatting in the top row as it's bolded by the platform

```suggestion
| Product support | Contact |
```

File: docs/resources/resources-links-help.md [hunk 2]
```diff
@@ -1,25 +1,76 @@
...
+| **Azure help and support** | If you have a paid Azure subscription, visit the Azure Admin Center to [open a ticket](https://ms.portal.azure.com/#blade/Microsoft_Azure_Support/HelpAndSupportBlade/newsupportreq). |
+| **General questions about Microsoft 365** | If you have general questions about Microsoft 365 or Office, submit your questions on [Microsoft 365 and Office Community](https://answers.microsoft.com/msoffice/forum).|
+
+## Developer community help
+
+Our community of developers use Stack Overflow, GitHub, and Microsoft Q&A to connect with other developers to ideate, get clarifications, and submit queries.
+
+### Developer community forums
+
+Post your questions and help other community members by sharing and responding to Teams App Development questions.
```
Comment (ElizabethSamuel-MSFT) [inline docs/resources/resources-links-help.md:28]:
```suggestion
Post your questions and help other community members by sharing and responding to Office Add-in Development questions.
```

File: docs/resources/resources-links-help.md [hunk 3]
```diff
@@ -1,25 +1,76 @@
...
+
+| **Developer need**|   **Contact**  |
```
Comment (ElizabethSamuel-MSFT) [inline docs/resources/resources-links-help.md:30]:
```suggestion
| Developer need | Contact |
```

File: docs/resources/resources-links-help.md [hunk 4]
```diff
@@ -1,25 +1,76 @@
...
+| **Documentation gaps** | If the documentation is missing something that would help you understand how to create add-ins, please submit an issue to the [Office Add-ins documentation GitHub repository](https://github.com/officedev/office-js-docs-pr/issues)|
+
+### Report issues
+
+Submit issues and ask general questions related to add-ins, documentation, and samples using the community help channels.
+
+| **Issue type** | **Contact** |
```
Comment (ElizabethSamuel-MSFT) [inline docs/resources/resources-links-help.md:39]:
```suggestion
| Issue type | Contact |
```

File: docs/resources/resources-links-help.md [hunk 5]
```diff
@@ -1,25 +1,76 @@
...
+| **Support need** | **Contact** |
+|:----------------------|:------------|
+|**Office Add-ins or Office JavaScript API feature requests**| To submit a feature request for the Office JavaScript API, please post your idea to the [Microsoft 365 Developer Platform Tech Community](https://techcommunity.microsoft.com/t5/microsoft-365-developer-platform/idb-p/Microsoft365DeveloperPlatform).|
+| **General questions** | Send general questions about the Office add-in platform to [Office Add-in Community Help](mailto:officeaddin. We encourage posting questions on the channels mentioned here and using email only if no other mode of communication is applicable. |
+
+## External learning resources
+
+The following books and classes provide add-ins information from a different perspective.
+
+|**Resource**|**More information**|
```
Comment (ElizabethSamuel-MSFT) [inline docs/resources/resources-links-help.md:67]:
```suggestion
|Resource|More information|
```

File: docs/resources/resources-links-help.md [hunk 6]
```diff
@@ -1,25 +1,76 @@
...
+| **Documentation gaps** | If the documentation is missing something that would help you understand how to create add-ins, please submit an issue to the [Office Add-ins documentation GitHub repository](https://github.com/officedev/office-js-docs-pr/issues)|
+
+### Report issues
+
+Submit issues and ask general questions related to add-ins, documentation, and samples using the community help channels.
+
+| **Issue type** | **Contact** |
+|:----------------------|:------------|
+| **Office Add-ins or Office JavaScript API issues** | If you encounter an issue (bug) with the Office JavaScript API, please submit an issue to the [Office JavaScript APIs GitHub repository](https://github.com/officedev/office-js/issues)|
+|**Documentation issues**| If you encounter an issue (bug) with the documentation, please submit an issue to the [Office Add-ins documentation GitHub repository](https://github.com/officedev/office-js-docs-pr/issues)|
```
Comment (ElizabethSamuel-MSFT) [inline docs/resources/resources-links-help.md:42]:
```suggestion
| **Office Add-ins or Office JavaScript API issues** | If you encounter an issue (bug) with the Office JavaScript API, please submit an issue to the [Office JavaScript APIs GitHub repository](https://github.com/officedev/office-js/issues).|
|**Documentation issues**| If you encounter an issue (bug) with the documentation, please submit an issue to the [Office Add-ins documentation GitHub repository](https://github.com/officedev/office-js-docs-pr/issues).|
```

File: docs/resources/resources-links-help.md [hunk 7]
```diff
@@ -1,25 +1,76 @@
...
+| **Issues with code samples** | Submit issues with samples to the [Office Add-ins code samples](https://github.com/OfficeDev/Office-Add-in-samples) repository. |
+
+### Documentation feedback
+
+Submit documentation feedback or updates using the community help channels.
+
+| **Feedback type** | **Contact** |
+|:--------------------------|:--------------------------|
+|**Documentation issues**| If you encounter an issue (bug) with the documentation, please submit an issue to the [Office Add-ins documentation GitHub repository](https://github.com/officedev/office-js-docs-pr/issues)|
+|**Documentation updates**| To make changes to the documentation yourself, choose the **Edit** link on an article and submit a pull request to the [Office Add-ins documentation GitHub repository](https://github.com/officedev/office-js-docs-pr)|
```
Comment (ElizabethSamuel-MSFT) [inline docs/resources/resources-links-help.md:52]:
```suggestion
|**Documentation issues**| If you encounter an issue (bug) with the documentation, please submit an issue to the [Office Add-ins documentation GitHub repository](https://github.com/officedev/office-js-docs-pr/issues).|
|**Documentation updates**| To make changes to the documentation yourself, choose the **Edit** link on an article and submit a pull request to the [Office Add-ins documentation GitHub repository](https://github.com/officedev/office-js-docs-pr).|
```

File: docs/resources/resources-links-help.md [hunk 8]
```diff
@@ -1,25 +1,76 @@
...
+
+### Feature request and general help
+
+Suggest a feature or vote up existing feature requests.
+
+| **Support need** | **Contact** |
+|:----------------------|:------------|
+|**Office Add-ins or Office JavaScript API feature requests**| To submit a feature request for the Office JavaScript API, please post your idea to the [Microsoft 365 Developer Platform Tech Community](https://techcommunity.microsoft.com/t5/microsoft-365-developer-platform/idb-p/Microsoft365DeveloperPlatform).|
+| **General questions** | Send general questions about the Office add-in platform to [Office Add-in Community Help](mailto:officeaddin. We encourage posting questions on the channels mentioned here and using email only if no other mode of communication is applicable. |
```
Comment (ElizabethSamuel-MSFT) [inline docs/resources/resources-links-help.md:61]: Is "mailto:officeaddin" the right destination for the Community Help link? Does it need to be `mailto:officeaddin@microsoft.com` or something like that?

```suggestion
| **General questions** | Send general questions about the Office Add-ins platform to [Office Add-ins Community Help](mailto:officeaddin). We encourage posting questions on the channels mentioned here and using email only if no other mode of communication is applicable. |
```

**Review by ElizabethSamuel-MSFT — APPROVED** [review]
Looks good. Just need the Community Help contact before merging.

**Pull request merged**